### DISCUSSION TITLE
Installer just closes after selecting options

### DISCUSSION THREAD

**Discussion by Speshals** [OpenVoiceOS/ovos-buildroot#272] [Q&A] [answered]
Get to just after confirming options and then the installer closes with no error or log. 
This is what is in the console: ➤ Checking for existing instance... [done]
➤ Retrieving OS information... [done]
➤ Detecting AVX/SIMD support... [done]
➤ Checking for Raspberry Pi board... [done]
➤ Detecting sound server... [done]
➤ Detecting display server... [done]
➤ Validating installer package requirements... [done]
➤ Creating installer Python virtualenv... [done]
➤ Installing Ansible requirements in Python virtualenv... [done]
➤ Looking for automated scenario... [done]
➤ Scan I2C bus for hardware auto-detection...[done]

**Comment by Speshals** [comment]
Installing on Raspberry Pi 5 and Raspberry OS Lite 64bit

**Reply by goldyfruit** [reply, marked as answer]
Issue opened https://github.com/OpenVoiceOS/ovos-installer/issues/100

**Comment by kamicavi** [comment]
Have you tried making the terminal window full screen before running the installer? I had the same problem, because I had SSH'ed in from my laptop and had the terminal running in a quarter of the screen. This sqaushed the TUI enough that the options at the very end of the installer (to install or quit) were pushed off the screen.
